src/backend/access/appendonly/appendonly_blkdir_udf.c

```diff
@@ -40,7 +40,7 @@ Datum gp_aoblkdir(PG_FUNCTION_ARGS);
 Datum
 gp_aoblkdir(PG_FUNCTION_ARGS)
 {
-	Oid       	aoRelOid = PG_GETARG_OID(0);
+	Oid	aoRelOid;
```

Comment:
Why not revert and this change?

Reply:
```suggestion
	Oid       	aoRelOid;
```

Reply:
It was made by pgindent

Reply:
Greenplum does not have a uniform code style, so running pgindent for the entire file is not the best idea. Therefore, I think this change should be removed.

Reply:
Just to clarify before running new regression tests set and making new approvals - this line was affected by the PR, and that's why it was indented. Do you suggest to skip pgindent for this line and make it indented differently. Why?

Reply:
Sorry, I didn't notice that not only the formatting has changed. Then it's fine.